.travis.yml

```diff
@@ -46,6 +46,17 @@ matrix:
     env:
       - BUILD=snippet
       - BUILD_TYPE=Release
+  - os: linux
+    compiler: 'doxygen'
+    env:
+      - BUILD=dox
+    addons:
+      apt:
+        packages: ['clang']
+    before_install:
+       - wget ftp://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.14.linux.bin.tar.gz
+       - tar -zxvf doxygen-1.8.14.linux.bin.tar.gz
+       - PATH=$PATH:`pwd`/doxygen-1.8.14/bin/
```

Comment:
it's not in the repos?

Reply:
I tried the standard repo one, but it is doxygen-1.8.6 and heavily fails to parse our code. (its from 2010; pre c++11 time)

Then I tried to find some more recent ones, but failed. Then I found how rapidjson does this and they simply download it and install it. https://github.com/Tencent/rapidjson/blob/master/travis-doxygen.sh

Reply:
Ok!